repm/types.go

```diff
@@ -86,3 +94,25 @@ type PullProgressResponse struct {
 	BytesReceived uint64 `json:"bytesReceived"`
 	BytesExpected uint64 `json:"bytesExpected"`
 }
+
+type BeginTransactionRequest struct {
+}
+
+type BeginTransactionResponse struct {
+	TransactionID int `json:"transactionId"`
+}
+
+type CloseTransactionRequest TransactionRequest
```

Comment:
Why distinguish between `CloseTransaction` and `Abort`/`Commit`? Seems like `Abort` and `Close` are the same thing.

Reply:
Yes. They are the same thing. I was thinking from a classical DB terminology with BEGIN COMMIT/ROLLBACK and it felt strange.

Reply:
Probably internally databases look more like what's happening here.